Python/CosmosDB-NoSQL_VectorSearch/CosmosDB-NoSQL-Quickstart-RAG-Chatbot.ipynb

```diff
@@ -0,0 +1,647 @@
+    "\n",
+    "# Create the Azure Cosmos DB for NoSQL async client for faster data loading\n",
+    "cosmos_async_client = CosmosClient(url=cosmos_conn, credential=cosmos_key)\n",
+    "\n",
+    "openai_endpoint = config['openai_endpoint']\n",
+    "openai_key = config['openai_key']\n",
+    "openai_api_version = config['openai_api_version']\n",
+    "openai_embeddings_deployment = config['openai_embeddings_deployment']\n",
+    "openai_embeddings_dimensions = int(config['openai_embeddings_dimensions'])\n",
+    "openai_completions_deployment = config['openai_completions_deployment']\n",
+    "\n",
+    "# Movies file url\n",
+    "storage_file_url = config['storage_file_url']\n",
+    "\n",
+    "# Create the OpenAI client\n",
+    "openai_client = AzureOpenAI(azure_endpoint=openai_endpoint, api_key=openai_key, api_version=openai_api_version)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "011ec9d5-8fe7-406f-85a7-027b84a5189f",
+   "metadata": {},
+   "source": [
+    "#  Create a database and containers with vector policies\n",
+    "\n",
+    "This function takes a database object, a collection name, the name of the document property that will store vectors, and the number of vector dimensions used for the embeddings."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "305807f8-0205-481a-9d71-e8b88b504019",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "db = await cosmos_async_client.create_database_if_not_exists(cosmos_database)\n",
+    "\n",
+    "# Create the vector embedding policy to specify vector details\n",
+    "vector_embedding_policy = {\n",
+    "    \"vectorEmbeddings\": [ \n",
+    "        { \n",
+    "            \"path\":\"/\" + cosmos_vector_property,\n",
+    "            \"dataType\":\"float32\",\n",
+    "            \"distanceFunction\":\"dotproduct\",\n",
+    "            \"dimensions\":openai_embeddings_dimensions\n",
+    "        }, \n",
+    "    ]\n",
+    "}\n",
+    "\n",
+    "# Create the vector index policy to specify vector details\n",
+    "indexing_policy = {\n",
+    "    \"vectorIndexes\": [ \n",
+    "        {\n",
+    "            \"path\": \"/\"+cosmos_vector_property, \n",
+    "            \"type\": \"quantizedFlat\" \n",
+    "        }\n",
+    "    ]\n",
+    "} \n",
+    "\n",
+    "\n",
+    "# Create the data collection with vector index (note: this creates a container with 10000 RUs to allow fast data load)\n",
+    "try:\n",
+    "    container = await db.create_container_if_not_exists( id=cosmos_collection, \n",
+    "                                                  partition_key=PartitionKey(path='/id'), \n",
+    "                                                  vector_embedding_policy=vector_embedding_policy,\n",
+    "                                                  offer_throughput=10000) \n",
+    "    print('Container with id \\'{0}\\' created'.format(id)) \n",
```

Comment:
this 'id' value being printed is the Python built-in id function - you probably want the container's actual id right? both here (line 156) and on line 168
```suggestion
    "    print('Container with id \\'{0}\\' created'.format(container.id)) \n",
```